core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -237,6 +241,10 @@ object ConsoleProducer {
             .withRequiredArg
             .describedAs("producer_prop")
             .ofType(classOf[String])
+    val producerConfigOpt = parser.accepts("producer.config", "Producer config properties file.")
```

Comment:
I'd remove `producer-property`, but that would break existing users right? Regarding the naming, I don't have a strong opinion, but we have `producer.config` in `VerifiableProducer`, `MirrorMaker`, ProducerPerformance` and `KafkaMigrationTool`. And `consumer.config` in `ConsoleConsumer`, `ConsumerPerformance`, `MirrorMaker`, `KafkaMigrationTool`.